app/javascript/components/app.jsx

```diff
@@ -14,6 +14,7 @@ import { getMainDefinition } from 'apollo-utilities';
 import ActionCableLink from 'graphql-ruby-client/subscriptions/ActionCableLink';
 import { persistCache } from 'apollo-cache-persist';
 import WidgetController from '@components/widget-controller';
+import VersionCompare from '@components/deployedVersionCompare';
```

Comment:
Nit: this file name might be clearer if it follows the '-' pattern as opposed to camelCase